src/yaml/aslYamlLanguageService.ts

```diff
@@ -56,18 +89,14 @@ export const getLanguageService = function(params: LanguageServiceParams, schema
         textDocument: TextDocument
     ) {
         const yamlDocument: YAMLDocument = parseYAML(textDocument.getText())
-        const validationResult: any[] = []
+        const validationResult: Diagnostic[] = []
 
         for (const currentYAMLDoc of yamlDocument.documents) {
             const validation = await aslLanguageService.doValidation(textDocument, currentYAMLDoc)
-            const syd = (currentYAMLDoc as unknown) as SingleYAMLDocument
-            if (syd.errors.length > 0) {
-                validationResult.push(...syd.errors)
-            }
-            if (syd.warnings.length > 0) {
-                validationResult.push(...syd.warnings)
-            }
-
+            const syd = currentYAMLDoc
+            validationResult.push(
+                ...syd.errors.concat(syd.warnings).map(err => convertYAMLDiagnostic(err, textDocument)
+            ))
```

Comment:
Nit move one bracket up to previous line to close map.

Reply:
done